lib/common/client/middleware/response/raise_error.rb

```diff
@@ -25,21 +25,21 @@ def on_complete(env)
 
           def raise_error!
             if status&.between?(400, 599)
-              raise Common::Exceptions::BackendServiceException.new(service_specific_i18n_key, response_values)
+              raise Common::Exceptions::BackendServiceException.new(service_i18n_key, response_values, @status, @body)
```

Comment:
@omgitsbillryan Yes, I think maybe response_values could just as well be derived in BackendServerException class. I am not happy with what that class is right now and will probably revisit that as a larger refactor.